apps/shared/test/gbfs_metadata_test.exs

```diff
@@ -38,12 +38,31 @@ defmodule Transport.Shared.GBFSMetadataTest do
                },
                cors_header_value: "*",
                feed_timestamp_delay: _,
-               vehicle_types: ["bicycle"]
+               vehicle_types: ["bicycle"],
+               stats: %{
```

Comment:
Du coup ici il n’y a pas de feed vehicle_status, et du coup pas les entrées de stats liées, il faudrait le préciser en commentaire. 

Cette absence de feed vehicle_status, ça arrive souvent, je me rends pas compte ? Normalement, d’après le code, ça a dû déclencher une entrée dans les logs ("Cannot get GBFS vehicle_status details:"), tu peux le vérifier ?

Reply:
C'est précisé par le biais du titre du test, `for a stations feed`.

Il n'y a pas de flux `vehicle_status` dans le cas d'un système uniquement avec des stations et où le producteur ne détaille pas où se trouve chaque véhicule. `stations_status` permet d'indiquer la capacité/disponibilité de chaque station, par type de véhicules.

Ça arrive fréquemment en prod, tu peux exécuter :
```sql
select rm.metadata->'feeds', r.url, r.title
from resource_metadata rm
join resource r on r.id = rm.resource_id
where rm.inserted_at::date = '2024-11-13'
  and not ((rm.metadata->'feeds' ? 'vehicle_status') or (rm.metadata->'feeds' ? 'free_bike_status'))
```

> ça a dû déclencher une entrée dans les logs ("Cannot get GBFS vehicle_status details:"), tu peux le vérifier ?

Il n'y aura rien dans les logs car ce log est ajouté dans cette PR et n'existait pas avant